src/components/Header/index.js

```diff
@@ -16,41 +16,51 @@ import List from 'material-ui-icons/List'
-        let loginButton = <Button style={buttonStyle} onClick={() => this.props.login()}><FormattedMessage id="login"/></Button>
+        let loginButton = <Button style={buttonStyle} onClick={() => this.props.login()}><Person/><FormattedMessage id="login"/></Button>
         if(this.props.user) {
             loginButton = <Button style={buttonStyle} onClick={() => this.props.logout()}>{this.props.user.displayName}</Button>
         }
-
         return (
-            <Toolbar className="mui-toolbar">
-                <div>
-                    <Link to="/" className="title">
-                        <img className="title-image" src={cityOfHelsinkiLogo} alt="City Of Helsinki" />
-                        <div className="title-text">Linked Events</div>
-                    </Link>
-                </div>
-                <div className="navbar-links">
-                    <Button className="mui-flat-button" style={buttonStyle} onClick={() => this.props.routerPush('/')}><FormattedMessage id="organization-events"/><List/></Button>
-                    <Button className="mui-flat-button" style={buttonStyle} onClick={() => this.props.routerPush('/search')}><FormattedMessage id="search-events"/><Search/></Button>
-                    <Button className="mui-flat-button" style={buttonStyle} onClick={() => this.props.routerPush('/event/create/new')}><FormattedMessage id="create-event"/><Add/></Button>
-                </div>
-                <div>
-                    <Button className="mui-flat-button" style={{...buttonStyle,...verticalAlignMiddle}} onClick={() => this.props.routerPush('/help')}><HelpOutline/></Button>
-                    {loginButton}
+            <div>
+                <Toolbar className="mui-toolbar">
+                    <div>
+                        <Link to="/" className="title">
+                            <img className="title-image" src={cityOfHelsinkiLogo} alt="City Of Helsinki" />
+                        </Link>
+                    </div>
+                    <div>
+                        {loginButton}
+                    </div>
+                </Toolbar>
+                <div className="navlinksHolder" style={navstyle}>
+                    <div className="title-text" style={{'justify-self': 'center', color:'#1B914A', fontWeight:700, alignSelf:'center'}}>Linked Courses</div>
+                    <div className="navbar-links">
+                        <Button className="mui-flat-button" style={buttonStyle2} onClick={() => this.props.routerPush('/')}><FormattedMessage id="organization-events"/><List/></Button>
+                        <Button className="mui-flat-button" style={buttonStyle2} onClick={() => this.props.routerPush('/search')}><FormattedMessage id="search-events"/><Search/></Button>
+                        <Button className="mui-flat-button" style={{...buttonStyle2,...verticalAlignMiddle}} onClick={() => this.props.routerPush('/help')}><HelpOutline/> HelpOutline</Button>
```

Comment:
cleanup all inline style gonna be nice

Reply:
There seem to be a breakage perhaps in the webpack config that prevents some or all of local sass styles to work on the components , hence the reason I believe someone already started using inline styling and since the styling is not much , I decided to use it too, perhaps you can have a check @midori0507 since you are doing cleanups